crates/strategies/blend-strategies/src/liquidation_strategy.rs

```diff
@@ -851,4 +851,23 @@ impl BlendLiquidator {
 
         Ok(())
     }
+    // checks if we should fill the auction
+    fn assess_fill(&self, block: u32, pending_fill: &OngoingAuction) -> bool {
+        if self.force_fill {
+            if (pending_fill.auction_type == 0 || pending_fill.auction_type == 1)
+                && (block - pending_fill.block_submitted) >= 198
+            {
+                return true;
+            } else if pending_fill.auction_type == 2
+                && (block - pending_fill.block_submitted) >= 350
```

Comment:
I would use the block given on the `auction_data` field instead of the block_submitted. The block_submitted will be 0 before it is sent off to the action queue.